cloudtrail-lambda-dynamo-cdk/README.md

```diff
@@ -1,15 +1,14 @@
-# Check Resource Tag Compliance Using CloudTrail, DynamoDB, and Lambda
+# Check S3 Object Tag Compliance Using CloudTrail, DynamoDB, and Lambda
 
-This pattern shows how to leverage CloudTrail resource creation API calls to check for required tags and determine compliance. The resources used in this pattern include CloudTrail, S3, Lambda, and DynamoDB which are all deployed via CDK. From the CloudTrail logs stored in S3, the relevant resource creation events are populated into a DynamoDB table via Lambda. The items written into the table are then checked for the required tags to determine compliance. 
+This pattern shows how to leverage CloudTrail resource creation API calls to check for required S3 object tags and determine compliance. The resources used in this pattern include CloudTrail, S3, Lambda, and DynamoDB which are all deployed via CDK. From the CloudTrail logs stored in S3, PutObject events are populated into a DynamoDB table via Lambda. The items written into the table are then checked for the required tags to determine compliance. 
 
-Learn more about this pattern at Serverless Land Patterns:[Create an AWS account](https://serverlessland.com/patterns/cloudtrail-lambda-dynamodb-cdk)
+Learn more about this pattern at Serverless Land Patterns: << Add the live URL here >>
```

Comment:
Missing live url

Reply:
Added the live url, thank you! - Regarding the example-pattern.json conflict, I made changes to the pattern so that file needed to be changed with the updated description. I'm unable to click on 'resolve conflicts' here on github, how should I go about fixing the this?